### PR TITLE
Improve package.json

### PR DESCRIPTION
**Pull request by zdroid**
- Use shorter repository field
- Add bugs URL, just in case

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -74,10 +74,8 @@
     "temp": "^0.8.3",
     "webpack": "^2.1.0-beta.25"
   },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/yarnpkg/yarn.git"
-  },
+  "repository": "yarnpkg/yarn",
+  "bugs": "https://github.com/yarnpkg/yarn/issues",
```
Comment (sindresorhus) [inline package.json]: This is not needed. It's inferred from the `repository` field.

https://github.com/npm/normalize-package-data#what-normalization-currently-entails

Comment (addyosmani) [inline package.json]: Yeah. This doesn't add anything that can't already be determined without it being present.

Comment (zdroid) [inline package.json]: Thanks @sindresorhus. I fixed that. :smile:

**Comment by Daniel15** [comment]
The changes to the binary files (eg. `__tests__/fixtures/add/install-deterministic/mirror-for-offline/mime-db-1.0.3.tgz`) seem unrelated, could you please remove them from your pull request? :)

**Pull request closed without merging**